browser/js/dashboard/newDashboard/newDash.controller.js

```diff
@@ -0,0 +1,15 @@
+app.controller('newDashboardCtrl', function (userFactory,$state,$uibModalInstance, $scope) {
+	
+	// $scope.form = {
+	// 	name: dashbaord.name, 
+	// 	description: dashboard.description
+	// }
```

Comment:
I think it's the former. What we're merging into master should be finished code. I know FlashDash is in development right now. But for most of our jobs, we're going to be working on apps that are currently deployed. By merging with master we're saying, "this code is rock solid and ready to be deployed immediately." So it's good to get into the habit of getting rid of old code before making pull requests.